akka-docs/src/main/paradox/persistence-fsm.md

```diff
@@ -128,3 +126,87 @@ On recovery state data is initialized according to the latest available snapshot
 `applyEvent` method.
 
 
+## Migration to EventSourcedBehavior
+
+Persistent FSMs can be represented using @ref[Persistence Typed](typed/persistence.md). The data stored by Persistence FSM can be read by an @apidoc[EventSourcedBehavior]
+using a snapshot adapter and an event adapter. The adapters are required as Persistent FSM doesn't store snapshots and user data directly, it wraps them in 
```

Comment:
```suggestion
using a snapshot adapter and an event adapter. The adapters are required because Persistent FSM doesn't store snapshots and user data directly, but wraps them in 
```